bin/build-theme.js

```diff
@@ -1,22 +0,0 @@
-const path = require('path');
```

Comment:
@iwwwi I've refactored the code a little bit so that if we do:

```bash
node_modules/.bin/travix-ui-kit -c ./myDestForCss -j ./myDestForJs -t ./myThemeYamlFile.yml -w
```

it will detect changes on the yaml file and re-bundles the `themes.scss` which then re-triggers the bundling of the `ui-bundle.*` files.